dev-libs/libei/libei-1.0.0.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 2014-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+inherit meson optfeature python-any-r1 udev
+
+DESCRIPTION="Library for Emulated Input, primarily aimed at the Wayland stack"
+HOMEPAGE="https://gitlab.freedesktop.org/libinput/libei"
+SRC_URI="https://gitlab.freedesktop.org/libinput/${PN}/-/archive/${PV}/${P}.tar.bz2"
+SRC_URI+=" https://github.com/nemequ/munit/archive/fbbdf1467eb0d04a6ee465def2e529e4c87f2118.tar.gz -> munit-9999.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/10"
```

Comment:
I don't see anything in the package that indicates `10` is the correct subslot. I don't even know that we need a subslot.